src/content/docs/installation/boot_managers.md

```diff
@@ -18,6 +18,25 @@ handles the task of loading an OS kernel into memory, often along with support f
 For a more detailed explanation and the reason both terms are often used interchangeably,
 please see "[Managing EFI Boot Loaders for Linux: Basic Principles](https://www.rodsbooks.com/efi-bootloaders/principles.html)" by Rod Smith.
 
+Here is the feature comparison table for the boot managers, accompanied by additional details and recommendations:
+
+### **Feature Comparison Table**
+
+| **Feature**                | **systemd-boot**                     | **rEFInd**                            | **GRUB**                           |
+|----------------------------|---------------------------------------|---------------------------------------|------------------------------------|
+| **UEFI Support**            | ✅ Yes                               | ✅ Yes                               | ✅ Yes                            |
+| **BIOS Support**            | ❌ No                                | ❌ No                                | ✅ Yes                            |
+| **Autodetection**           | ❌ Limited (manual configuration)    | ✅ Full autodetection                | ✅ Autodetection (regenerates config) |
+| **Theming/Customization**   | ❌ Minimal                           | ✅ Extensive                          | ✅ Extensive                      |
+| **Configuration Complexity**| Simple                              | Moderate                             | Complex                          |
+| **Secure Boot Support**     | ✅ Yes                               | ✅ Yes                               | ✅ Yes                            |
+| **Filesystem Support**      | FAT, FAT16, FAT32                   | FAT, EXT4, BTRFS                     | Most Linux FS                     |
+| **Encrypted Boot**          | ❌ No                                | ❌ No                                | ✅ Yes (boot partition)           |
+| **Boot Time**               | Fastest                             | Moderate                             | Slowest                          |
+| **Hardware Compatibility**  | UEFI-only                          | UEFI-only                           | UEFI and BIOS                    |
+| **Ease of Setup**           | Moderate                            | Easy                                 | Moderate                        |
```

Comment:
This can "bait" users into thinking that the other boot managers except rEEFInd is "hard" to setup while it's not true. Can you give cases where GRUB and sd-boot are harder to setup than rEFInd? In the context of CachyOS, systemd-boot is complemented with `systemd-boot-manager` which aids in creating new entries for the each kernel installed/updated and a generic way to configure entries from one file (similar to GRUB).

Reply:
Personally, I have sometimes struggled to configure and maintain GRUB and ended up with a broken system. That's why I wrote this. SystemD booting is easy via systemd-boot-manager, true.

I would say we can also remove this section, because the diffculty of configure this up is marginal?

Reply:
> I would say we can also remove this section

Instead of removing it altogether, I think replacing it with "Partition Table" would be better. In the context of setting up boot managers, all of it is handled by the installer. The only case where a user would have to do it themselves is through manual partitioning, in which case adding a partition table comparison would make sense. Consider https://wiki.cachyos.org/installation/installation_on_root/#manual-partitioning No. 5 for that. (Note that these partition tables aren't strict requirements[1][2], it's just what CachyOS uses.)

[1] rEFInd used to have a three-partition table, `/boot`, `/boot/efi` and `/`
[2] It is also possible for `systemd-boot` to have separate ESP (EFI binaries) and boot partitions (kernel image and initramfs).